chema-cabeza/Precurso/Airlines/airlines-pro.js

```diff
@@ -0,0 +1,323 @@
+let flights = [
+
+    { id: 00, to: 'Bilbao', from: 'Barcelona', cost: 1600, scale: false },
+
+    { id: 01, to: 'New York', from: 'Barcelona', cost: 700, scale: false },
+
+    { id: 02, to: 'Los Angeles', from: 'Madrid', cost: 1100, scale: true },
+
+    { id: 03, to: 'Paris', from: 'Barcelona', cost: 210, scale: false },
+
+    { id: 04, to: 'Roma', from: 'Barcelona', cost: 150, scale: false },
+
+    { id: 05, to: 'London', from: 'Madrid', cost: 200, scale: false },
+
+    { id: 06, to: 'Madrid', from: 'Barcelona', cost: 90, scale: false },
+
+    { id: 07, to: 'Tokyo', from: 'Madrid', cost: 1500, scale: true },
+
+    { id: 08, to: 'Shangai', from: 'Barcelona', cost: 800, scale: true },
+
+    { id: 09, to: 'Sydney', from: 'Barcelona', cost: 150, scale: true },
+
+    { id: 10, to: 'Tel-Aviv', from: 'Madrid', cost: 150, scale: false } ];
+
+let fifteen = 15;
+
+(function skylabAirlines () {
+
+
+let user = prompt (`Introduce user name, please:`);
+
+    if (user === null) {console.log ("Thanks, bye!"); return false;};
+
+let media = 0;
+let numScales = 0;
+
+console.log(`Welcome to Skylab Airlines, ${user}`);
+
+for (let i = 0; i < flights.length; i++) {
+
+    if (flights[i].scale) { 
+
+        console.log (`(ID: ${flights[i].id}) Flight with origin: ${flights[i].from} and destination: ${flights[i].to} has a cost of ${flights[i].cost}€ and has scales`);
+        numScales ++;
+    }
+    else {console.log (`(ID: ${flights[i].id}) Flight with origin: ${flights[i].from} and destination: ${flights[i].to} has a cost of ${flights[i].cost}€ and doesn't have any scale`);}
+    
+    media += flights[i].cost;
+}
+
+console.log(`The average fare for the flights is: ${(media/flights.length).toFixed(2)}€`);
+console.log(`The number of flights with scale is: ${numScales}`);
+
+lastFlights();
+
+adminQuestion();
+
+})()
+
+function adminQuestion (question = `¿Are you ADMIN? Y/N`){
```

Comment:
En este caso el argument te lo puedes ahorrar y asignas a answer = prompt(`¿Are you ADMIN? Y/N`)